package.json

```diff
@@ -3,8 +3,8 @@
     "private": true,
     "version": "0.0.0",
     "scripts": {
-        "dev": "ENV=DEVELOPMENT nodemon -w src/ -x tsx src/app.ts",
```

Comment:
go ahead and put an ENV variable into your `.env` file! that's probably a cleaner way of doing this for now.